src/helm/benchmark/metrics/metric.py

```diff
@@ -88,6 +89,19 @@ def process(self, request_state_set: RequestStateSet) -> List[Stat]:
                 )
             )
 
+        # Drop any instance stats that contain NaN, because Python's stdlib json.dumps()
```

Comment:
Only question is whether this is the right place to put it.  I think maybe we should at least make this a helper function?

Reply:
Seems like it could be better to put it closer to the point of serialization?

Reply:
We should make it clear that upstream computation should not produce NaNs, and this is just a bandaid safety check?  (In which case, we should fix things upstream.)

Reply:
OK, I moved it to the point of serialization. I'll do the upstream fix in a different PR.